cmd/util.go

```diff
@@ -360,3 +360,15 @@ func prettifyReason(reason string) string {
 	}
 	return reason
 }
+
+func isPSCDefinedCSCUndefined(podSpec PodSpecV1, container ContainerV1) bool {
```

Comment:
I thought u want to check if container SecurityContext is define properly or else use pod securityContext ? from #138 it says that SecurityContext is inherited from pod, so does it means that if container security context is defined then we could audit container else we just audit pod security context?

Reply:
ContainerSC wins over PodSC. But I left a longer explanation on one of the tests.

Reply:
it actually took me a while to fully understand what your function is try doing. I guess if you want to improve the readability you can change the function to check if isCSCDefined() eg), and u can just check for if container.ServurityConx != nil && contain.SecurityConx.RunAsNonRoot != nil {return true ; else false}

Reply:
We actually also need to check if PodSpec.SC != nil && PodSpec.SC.RANAR != nil, this is because by default we should point this error to be a Container SC error rather than it being a Pod SC error, this is solely because CSC takes precedence, but yes I have changed up the function a bit so it's looks cleaner 
```
func isCSCWellDefined(podSpec PodSpecV1, container ContainerV1) bool {
	if container.SecurityContext != nil && container.SecurityContext.RunAsNonRoot != nil {
		return true
	}
	if podSpec.SecurityContext == nil || podSpec.SecurityContext.RunAsNonRoot == nil {
		return true
	}
	return false
}
```
What do you think?